src/apps/pillarx-app/components/WalletPortfolioButtons/WalletPortfolioButtons.tsx

```
@@ -15,12 +15,15 @@ import ReceiveModal from '../ReceiveModal/ReceiveModal';
 import BodySmall from '../Typography/BodySmall';
 import WalletConnectDropdown from '../WalletConnectDropdown/WalletConnectDropdown';
 
+// hooks
+import useTransactionKit from '../../../../hooks/useTransactionKit';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTransactionKit'. Consider to delete it or rename it.

'useTransactionKit' is defined here:


(lint/suspicious/noRedeclare)